5.x/crud-filters.md

```diff
@@ -25,6 +25,10 @@ $this->crud->removeFilter($name);
 $this->crud->removeAllFilters();
 
 $this->crud->filters(); // gets all the filters
+
+// aditionally you should be able to change Filter attributes by appending `->methods` in your filter definition. Eg:
+$this->crud->addFilter($options, $values, $filter_logic)->type('number');
+// this would set the filter attribute `type` to be equal to `number`. 
```

Comment:
```suggestion
// aditionally you should be able to change Filter attributes by appending `->methods` in your filter definition. Eg - this would set the filter attribute `type` to be equal to `number`:
$this->crud->addFilter($options, $values, $filter_logic)->type('number');
```